### ISSUE TITLE
Npc Damage

### ISSUE THREAD

**Issue by Alf21** [Open-GTO/FCNPC#5]
I dont know whether its a bug or not implemented, FCNPCs can't damage other FCNPCs.
I coded a bodyguard NPC and a zombie NPC and let them fight, but there does not any damage appear and the onTakeDamage event was not called.

And big thank for fixing this project! :)

**Comment by kurta999** [comment]
http://forum.sa-mp.com/showthread.php?t=428066&page=250

**Comment by ziggi** [comment]
I tried to fix it, but I did not succeed.
@kurta999, SendBulletData from YSF does not work for me:
i try this:
```Pawn
SendBulletData(playerid, playerid, BULLET_HIT_TYPE_PLAYER, GetPlayerWeapon(playerid), x, y, z, 0.0, 0.0, 0.0, 0.1, 0.1, 0.1);
```
and this (with disabling IsPlayerConnectedEx check in YSF):
```Pawn
if (!strcmp(cmdtext, "/shootme")) {
	new Float:x, Float:y, Float:z;
	GetPlayerPos(npc, x, y, z);

	new Float:tx, Float:ty, Float:tz;
	GetPlayerPos(playerid, tx, ty, tz);

	SendBulletData(npc, playerid, BULLET_HIT_TYPE_PLAYER, GetPlayerWeapon(playerid), x, y, z, tx, ty, tz, 0.1, 0.1, 0.1);
	return 1;
}
```

**Comment by Alf21** [comment]
maybe change to FCNPC_GetWeapon(npc) and FCNPC_GetPosition(npc) ?

if (!strcmp(cmdtext, "/shootme")) {
    new Float:x, Float:y, Float:z;
    FCNPC_GetPosition(npc, x, y, z);

    new Float:tx, Float:ty, Float:tz;
    GetPlayerPos(playerid, tx, ty, tz);

    SendBulletData(npc, playerid, BULLET_HIT_TYPE_PLAYER, FCNPC_GetWeapon(playerid), x, y, z, tx, ty, tz, 0.1, 0.1, 0.1);
    return 1;
}

**Comment by ziggi** [comment]
@Alf21, no, just SendBulletData on Linux does not working.
Damage has been fixed here (works only on Windows): 24f058fc0b88a160fd0847f48034b3856985f5ee

**Comment by kurta999** [comment]
I'll check that SendBulletData why not work at linux soon.
